pymc3/tests/test_models_utils.py

```diff
@@ -51,6 +50,8 @@ def test_dict_input(self):
         m, l = utils.any_to_tensor_and_labels(self.data.to_dict("list"))
         self.assertMatrixLabels(m, l, mt=self.data[l].values, lt=l)
 
+    @pytest.mark.xfail
```

Comment:
Can we dig the reason for this one?

Reply:
I think that `pymc3.glm.utils.any_to_tensor_and_labels` doesn't support converting data structures containing TensorType (mostly because `theano.tensor.as_tensor_variable` called on line 132 doesn't).

I haven't been able to understand what conditions made this test pass. I guess that if we need `pymc3.glm.utils.any_to_tensor_and_labels` to be able to handle data structure containing tensor objects we should probably change that

Reply:
I tracked it down to this commit: https://github.com/pymc-devs/pymc3/commit/4f67d639f984195f81b03cbfc19de64507f9f1f1

@rpgoldman could you chime in? What is the source of this failure? It would be helpful to have a more clear reason specified in the xfail.

Reply:
The xfail was there before, so this shouldn't block us from moving forward with this PR.